docs/index.adoc

```diff
@@ -6443,6 +6451,109 @@ class Sub implements Runnable {
 }
 ----
 
+.Kotlin
+[source,kotlin,role="secondary"]
+----
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.apache.logging.log4j.core.config.Configurator
+
+import picocli.CommandLine
+import picocli.CommandLine.Command
+import picocli.CommandLine.Mixin
+import picocli.CommandLine.Model.CommandSpec
+import picocli.CommandLine.Option
+import picocli.CommandLine.ParseResult
+import picocli.CommandLine.RunLast
+import picocli.CommandLine.Spec
+
+class LoggingMixin {
+    @Spec(Spec.Target.MIXEE)
+    private lateinit var mixee : CommandSpec// spec of the command where the @Mixin is used
+    var verbosity = BooleanArray(0)
+
+    /**
+     * Sets the specified verbosity on the LoggingMixin of the top-level command.
+     * @param verbosity the new verbosity value
+     */
+    @Option(
+        names = ["-v", "--verbose"], description = [
+                 "Specify multiple -v options to increase verbosity.",
+                 "For example, `-v -v -v` or `-vvv`"]
+    )
+    fun setVerbose(verbosity: BooleanArray) {
+        // Each subcommand that mixes in the LoggingMixin has its own instance
+        // of this class, so there may be many LoggingMixin instances.
+        // We want to store the verbosity value in a single, central place,
+        // so we find the top-level command,
+        // and store the verbosity level on our top-level command's LoggingMixin.
+        (mixee.root().userObject() as MyApp).loggingMixin.verbosity = verbosity
+    }
+}
+
+@CommandLine.Command(name = "app", subcommands = [Sub::class])
```

Comment:
I prefer `@Command` to `@CommandLine.Command`

Reply:
I overlooked that. I prefer @Command to @CommandLine.Command, too